dts/arm/nxp/nxp_rt1024.dtsi

```diff
@@ -42,13 +42,13 @@
 	status = "okay";
 	reg = <0x402a8000 0x4000>, <0x60000000 DT_SIZE_M(4)>;
 	/* 32 megabit internal flash present on chip */
-	w25q32jvwj0: w25q32jvwj0@0 {
+	w25q32jvwj0: w25q32jvwj@0 {
 		compatible = "nxp,imx-flexspi-nor";
-		size = <DT_SIZE_M(32)>;
+		size = <DT_SIZE_M(4 * 8)>;
```

Comment:
Why?

Reply:
> Why?

For better reading, to emphasizes that this value is bits, 4 Megabytes not 32.

Reply:
Ok that's what I figured